js/app.js

```diff
@@ -0,0 +1,54 @@
+import itemsApi from './component-todo-api.js'; 
+import html from './html.js'; 
+import AddItem from './component-add-todo.js';
+import TodoList from './component-todo-list.js';
+
+const items = itemsApi.getAll();
+console.log('hello', items);
+
+function makeTemplate() {
+    return html`
+    <form id="add-form">
```

Comment:
```suggestion
    <form id="add-form">
            <label>
                
            </label>
            <label>
                
            </label>
            <label>
                
            </label>
        </form>
        <h2> List </h2>
        <ul id="list"></ul>
```
Try to properly nest child elements in your templates much the same way you would in your HTML.